MC/bin/o2dpg_sim_workflow.py

```diff
@@ -83,6 +83,7 @@
 parser.add_argument('-colBkg',help='embedding background collision system', default='PbPb')
 
 parser.add_argument('-e',help='simengine', default='TGeant4')
+parser.add_argument('--noGeant',help='prohibits any Geant transport/physics', action='store_true')
```

Comment:
I think it is in principle fine although this option seems to be added because you would like to do some studies? In such a case, all following steps do not make sense.
But if this is a global option here, we might want to include some more logic and not even define the digitisation and reco tasks.
What do you think @sawenzel ?